integration-tests/utils/integration-utils.js

```diff
@@ -244,7 +246,7 @@ const withScreenshot = _.curry((testName, fn) => async options => {
 })
 
 const logPageConsoleMessages = page => {
-  const handle = msg => console.log('page.console', msg.text(), msg)
+  const handle = msg => rawConsole.log('page.console', msg.text(), msg)
```

Comment:
No longer using this method (in fact, nobody is), but changing to `rawConsole` for the future.